package.json

```diff
@@ -52,27 +52,27 @@
   ],
   "devDependencies": {
     "@babel/core": "^7.9.0",
-    "@babel/preset-env": "^7.9.0",
+    "@babel/preset-env": "^7.9.5",
     "@babel/register": "^7.9.0",
-    "@rollup/plugin-commonjs": "^11.0.2",
+    "@rollup/plugin-commonjs": "^11.1.0",
     "@rollup/plugin-json": "^4.0.2",
-    "@rollup/plugin-node-resolve": "^7.1.1",
+    "@rollup/plugin-node-resolve": "^7.1.3",
     "babel-plugin-transform-es2017-object-entries": "0.0.5",
     "chai": "^4.2.0",
     "eslint": "^6.8.0",
     "esprima": "~4.0.1",
     "mocha": "^7.1.1",
-    "nyc": "^15.0.0",
+    "nyc": "^15.0.1",
     "pegjs": "~0.10.0",
-    "rollup": "^1.32.0",
+    "rollup": "^1.32.1",
```

Comment:
Why is it useful to bump the patch version when using `^`? My understanding is that that implies we *require* `1.32.1`, and `1.32.0` would be insufficient. But that is not the case.

Reply:
As I see it, it would be an advantage to trigger (compatible) updates downstream for buggy versions--no less useful for devDependencies where one benefits from everyone being on the same page (or, with the package lock not being used, at least free of presumably buggier versions).

I think users often inadvertently use buggy versions because they haven't happened to have run `npm update` (or don't even know about this npm feature) rather than because they are unwilling to automatically get the benefits of improved versions upon a new install. As long as we have confirmed those latest versions, and we are not passing on major bumps without bumping our own version accordingly, it seems useful to me to pass that on to consumers, not to mention avoid bug reports based on usage of outdated versions.

Perhaps, this was the case with the issue this PR addresses, since the estraverse version wasn't even out of range and could have been gotten with an `npm update`. OTOH, it could have been a midstream consumer who just appreciated not having to field bug reports from those further downstream.